src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDay = 40;
+  const dayPrice = oneDay * days;
+  let discount = 0;
+
+  if (days >= 0 && days <= 2) {
+    return dayPrice;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+
+    return dayPrice - discount;
+  } else if (days >= 3) {
```

Comment:
The condition `days >= 7` correctly applies a $50 discount, but the `else if (days >= 3)` condition will also be true for days >= 7. Consider rearranging the conditions to avoid overlapping and ensure the correct discount is applied.